packages/api-electron/index.js

```diff
@@ -13,4 +16,11 @@ module.exports = () => {
       text: msg.options.body
     });
   });
+
+  ipc.on('ssf-new-window', (e, msg) => {
+    const newWin = new BrowserWindow();
+    newWin.loadURL(msg.url);
+    windows.push(newWin);
```

Comment:
Is this a potential memory leak, if the window isn't removed from `windows` when it's closed?

Could we remove the closed window reference from `windows` when its `close` or `closed` event is fired?